README.md

```diff
@@ -113,3 +113,6 @@ To get strings translated, we use the Zanata, the web-based translation platform
     grunt potupload
 
 Prior to do that, please make sure you have an account on the Zanata server. if not, please visit https://translate.zanata.org and follow up the steps at http://docs.zanata.org/en/release/user-guide/account/account-sign-up/ to create an account, and http://zanata-client.readthedocs.io/en/latest/configuration/ to store the API key into $HOME/.config/zanata.ini.
+
+## Node version :
+FHC requires node version: >=0.10 <= 4.4.0
```

Comment:
I would specify that fh-fhc works with 0.10 and 4.4.0 - because technically there is 0.12 version between which may work but we do not test our tooling against. Our unit tests are executed against this two targets.